### PR TITLE
[Yamato] Disabling mac for VFX LW project

### PR DESCRIPTION
**Pull request by sophiaaar**
### Purpose of this PR
Disabling the VFX Universal project from running on Mac because VFX is currently broken on Mac (spoke to Vlad to confirm this)

---
### Testing status
**Yamato**: confirmed disabling it worked
https://yamato.prd.cds.internal.unity3d.com/jobs/78-ScriptableRenderPipeline/tree/yamato%252Fdisable-vfx-mac

---
### Overall Product Risks
**Technical Risk**: None

**Halo Effect**: None

### CODE REVIEW

File: .yamato/upm-ci-vfx_lwrp.yml [hunk 1]
```diff
@@ -5,14 +5,14 @@ platforms:
...
     flavor: b1.large
-  - name: OSX
-    type: Unity::metal::macmini
-    image: slough-ops/macos-10.14-xcode
-    flavor: m1.mac
-  - name: OSX_OpenGLCore
-    type: Unity::VM::osx
-    image: buildfarm/mac:stable
-    flavor: m1.mac
+ # - name: OSX
```
Comment (JessAtUnity) [inline .yamato/upm-ci-vfx_lwrp.yml]: Should we have a comment for when the mac configuration should be restored?

Comment (sophiaaar) [reply .yamato/upm-ci-vfx_lwrp.yml]: done, thank you

**Comment by phi-lira** [comment]
Adding @PaulDemeulenaere for awareness.

**Comment by VladNeykov** [comment]
A fix was pushed, so the VFX tests for mac can be reactivated soon:
https://ono.unity3d.com/unity/unity/pull-request/91288/_/metal/editor

**Comment by sophiaaar** [comment]
Spoke to Vlad - the fix is on its way, so this PR can be closed

**Pull request closed without merging**